.appveyor.yml

```diff
@@ -5,6 +5,7 @@ cache:
 environment:
     matrix:
         - cmake_build_type: Debug
+          arch: x64
```

Comment:
There are so many "arch"es already, please rename to something foo_arch, like below.